src/main/scala/dpla/ingestion3/enrichments/LanguageEnrichment.scala

```diff
@@ -0,0 +1,112 @@
+package dpla.ingestion3.enrichments
+
+import dpla.ingestion3.model.SkosConcept
+import dpla.ingestion3.utils.FileLoader
+
+/**
+  * Language mapping examples
+  */
+class LanguageEnrichment extends FileLoader with VocabEnrichment[SkosConcept] {
+
+  // Files to source vocabulary from
+  private val fileList = Seq(
+    "/languages/iso639-2.csv",
+    "/languages/iso639-3.csv",
+    "/languages/dpla-lang.csv"
+  )
+
+  // performs term lookup
+  private val lookup = new VocabLookup[SkosConcept](
+    (term: SkosConcept) => normalizationFunc(term)
+  )
+
+  // combine two SkosConcepts
+  private val merger = new VocabMerge[SkosConcept](
+    (prov: SkosConcept, enriched: SkosConcept) => mergeFunc(prov, enriched)
+  )
+
+  /**
+    * Normalize providedLabel value for retrieval
+    *
+    * @param term SkosConcept
+    * @return String
+    */
+  private def normalizationFunc(term: SkosConcept): String = term
+      .providedLabel.getOrElse("")
+      .toLowerCase()
+      .trim
+
+  /**
+    * Merge provided and enriched values to preserve providedLabel
+    *
+    * @param prov Original value
+    * @param enriched Enriched form of `prov`
+    * @return Enriched SkosConcept with original value's providedLabel
+    */
+  private def mergeFunc(prov: SkosConcept, enriched: SkosConcept) =
+    enriched.copy(providedLabel = prov.providedLabel)
+
+  /**
+    * Read CSV files and load vocabulary into mappers
+    *
+    * @return
+    */
+  //noinspection TypeAnnotation,UnitMethodIsParameterless
```

Comment:
I don't understand what this comment means.

Reply:
That tells IntelliJ to not complain there is no return type specified for the method.